README.md

```diff
@@ -65,53 +65,53 @@ Below is the list of currently implemented methods, including the already availa
+| [KOS](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.classification.kos.KOS) | ✅ |
+| [MACE](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.classification.mace.MACE) | ✅ |
 | BCC | 🟡 |
 
 ### Multi-Label Responses
 
 |Method|Status|
 |-|:-:|
-|[Binary Relevance](reference/crowdkit.aggregation.multilabel.binary_relevance.BinaryRelevance.md)|✅|
+|[Binary Relevance](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.multilabel.binary_relevance.BinaryRelevance)|✅|
 
 ### Textual Responses
 
 | Method | Status |
 | ------------- | :-------------: |
-| [RASA](reference/crowdkit.aggregation.embeddings.rasa.RASA.md) | ✅ |
-| [HRRASA](reference/crowdkit.aggregation.embeddings.hrrasa.HRRASA.md) | ✅ |
-| [ROVER](reference/crowdkit.aggregation.texts.rover.ROVER.md) | ✅ |
+| [RASA](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.embeddings.rasa.RASA) | ✅ |
+| [HRRASA](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.embeddings.hrrasa.HRRASA) | ✅ |
+| [ROVER](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.texts.rover.ROVER) | ✅ |
 
 ### Image Segmentation
 
 | Method | Status |
 | ------------------ | :------------------: |
-| [Segmentation MV](reference/crowdkit.aggregation.image_segmentation.segmentation_majority_vote.SegmentationMajorityVote.md) | ✅ |
-| [Segmentation RASA](reference/crowdkit.aggregation.image_segmentation.segmentation_rasa.SegmentationRASA.md) | ✅ |
-| [Segmentation EM](reference/crowdkit.aggregation.image_segmentation.segmentation_em.SegmentationEM.md) | ✅ |
+| [Segmentation MV](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.image_segmentation.segmentation_majority_vote.SegmentationMajorityVote) | ✅ |
+| [Segmentation RASA](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.image_segmentation.segmentation_rasa.SegmentationRASA) | ✅ |
+| [Segmentation EM](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.image_segmentation.segmentation_em.SegmentationEM) | ✅ |
 
 ### Pairwise Comparisons
 
 | Method | Status |
 | -------------- | :---------------------: |
-| [Bradley-Terry](reference/crowdkit.aggregation.pairwise.bradley_terry.BradleyTerry.md) | ✅ |
-| [Noisy Bradley-Terry](reference/crowdkit.aggregation.pairwise.noisy_bt.NoisyBradleyTerry.md) | ✅ |
+| [Bradley-Terry](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.pairwise.bradley_terry.BradleyTerry) | ✅ |
+| [Noisy Bradley-Terry](https://toloka.ai/docs/crowd-kit/reference/crowdkit.aggregation.pairwise.noisy_bt.NoisyBradleyTerry) | ✅ |
 
 ### Learning from Crowds
 
 |Method|Status|
 |-|:-:|
-|[CrowdLayer](reference/crowdkit.learning.crowd_layer.CrowdLayer.md)|✅|
-|[CoNAL](reference/crowdkit.learning.conal.Conal.md)|✅|
+|[CrowdLayer](https://toloka.ai/docs/crowd-kit/reference/crowdkit.learning.crowd_layer.CrowdLayer)|✅|
+|[CoNAL](https://toloka.ai/docs/crowd-kit/reference/crowdkit.learning.conal.Conal)|✅|
```

Comment:
This one should be <https://toloka.ai/docs/crowd-kit/reference/crowdkit.learning.conal.CoNAL>.